pkg/controller/ttlafterfinished/ttlafterfinished_controller.go

```diff
@@ -131,15 +131,15 @@ func (tc *Controller) addJob(obj interface{}) {
 
 func (tc *Controller) updateJob(old, cur interface{}) {
 	job := cur.(*batch.Job)
-	klog.V(4).Infof("Updating job %s/%s", job.Namespace, job.Name)
+	klog.V(4).InfoS("Updating Job", "jobNamespace", job.Namespace, "job", job.Name)
 
 	if job.DeletionTimestamp == nil && needsCleanup(job) {
 		tc.enqueue(job)
 	}
 }
 
 func (tc *Controller) enqueue(job *batch.Job) {
-	klog.V(4).Infof("Add job %s/%s to cleanup", job.Namespace, job.Name)
+	klog.V(4).InfoS("Add Job to cleanup", "jobNamespace", job.Namespace, "job", job.Name)
```

Comment:
Same as above

Reply:
This can actually use `klog.KRef`, thanks for catching it.